macos.md

```diff
@@ -192,6 +192,7 @@ Make sure that you're running the 2nd-newest macOS version or the newest version
     ```bash
     [[ -d /opt/homebrew/var/postgresql@16 ]] && PGDATA_TMP=/opt/homebrew/var/postgresql@16 || PGDATA_TMP=/usr/local/var/postgresql@16
     echo "\nexport PGDATA=$PGDATA_TMP\nexport LC_ALL=en_US.UTF-8\nexport PSQL_PAGER=\"less --chop-long-lines --header 1\"" >> ~/`[[ $SHELL == *"zsh" ]] && echo '.zshrc' || echo '.bash_profile'`
+    sed -i '' "/^[[:space:]]*lc_messages[[:space:]]*=/ s/^#*.*$/lc_messages = 'en_US.UTF-8'/" "$PGDATA/postgresql.conf"
```

Comment:
Setting the `lc_messages` to utf-8 in macOS using sed

Reply:
can we simplify this regular expression? eg. the `[[:space:]]`, combining the 2 commands into 1

Reply:
Does this mean I can use the `[[:space:]]` thing?
It looks like this `\s` doesn't work as it seems some regex don't work with sed and if I find hat works, we may nee to turn on the `extended regular expression`  `-E` flag of sed

Reply:
we ended up switching to `perl` for this